lib/classes/logged_browser.rb

```diff
@@ -8,6 +8,6 @@ def call
   end
 
   def browser
-    @browser ||= Watir::Browser.new
+    @browser ||= Watir::Browser.new(:chrome)
```

Comment:
Also going to go ahead and merge, I think a good thing to do would be add this as a config option for which browser to use